utils_cv/action_recognition/model.py

```diff
@@ -3,10 +3,9 @@
 
 from collections import OrderedDict
 import os
-import time
 import warnings
 import numpy as np
-from typing import Union, Dict, Tuple, Any
+from typing import Union, Dict, Tuple, Any, List, Callable
```

Comment:
[minor] imports not fully alphabetically sorted

Reply:
done